### DISCUSSION TITLE
【1508740】轩辕剑2和【427030】轩辕剑外传穹之扉

### DISCUSSION THREAD

**Discussion by qq619230876** [pjy612/SteamManifestCache#99] [Share]
appid：1508740
keys：
```
1508741;5b56a4411827c0072421584eec11065101858b104ddfbf9aa7d524a7e5e7b367
```
appid：427030
keys：
```
427031;aff7172a7fe2cf4150333ff521bc8939d78594c7d7c13cbe76768aec462f8c88
```
邮箱：619230876@qq.com
code：B35BFDF0F79CE8B7
[轩辕剑2清单.zip](https://github.com/pjy612/SteamManifestCache/files/13719820/2.zip)
[轩辕剑穹之扉清单.zip](https://github.com/pjy612/SteamManifestCache/files/13719821/default.zip)

**Comment by pjy612** [comment]
查收
